integration-tests/src/test/fail-transfer.test.ts

```diff
@@ -26,7 +26,7 @@ describe('FailTransfer', (): void => {
     it('fee is min 0.01 CTC', async (): Promise<void> => {
         const { api } = ccApi;
         const transferId = createFundingTransferId(blockchain, '0xffffffffffffffffffffffffffffffffffffffff');
-        const cause = api.createType('PalletCreditcoinOcwErrorsVerificationFailureCause', 'TransferFailed');
+        const cause = api.createType('PalletCreditcoinOcwErrorsVerificationFailureCause', 'TaskFailed');
```

Comment:
We'll need integration test for `request_collect_coins`, `fail_collect_coins` and `persist_collect_coins` min fees.